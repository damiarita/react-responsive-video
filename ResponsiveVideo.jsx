import React from 'react';

const ResponsiveVideo = () => {
      return (
        <picture/>
      );
    }
  
  export default ResponsiveVideo;