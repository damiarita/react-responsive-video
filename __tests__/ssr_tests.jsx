import {create, act} from 'react-test-renderer';
import ResponsiveVideo from '../src'


it('SSR renders correctly', () => {
  const component = create(<ResponsiveVideo/>);
  expect(component.toJSON()).toMatchSnapshot();
});
