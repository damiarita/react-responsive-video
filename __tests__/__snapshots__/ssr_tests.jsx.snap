// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SSR renders correctly 1`] = `
<picture
  data-env="ssr"
  data-inner={1024}
  data-width={0}
/>
`;
